### ISSUE TITLE
issue of training stage~

### ISSUE THREAD

**Issue by ZHONIN** [facebookresearch/HighResCanopyHeight#9]
Hello, thank you for sharing such excellent work. Additionally, will you and your team be making the training code publicly available for learning and reproducing your outstanding work? （including the training scripts and loss functions for both phase one self-supervised training and phase two supervised training processes. : )  Looking forward to your response.

**Comment by JohnMBrandt** [comment]
Hello -- the training code is adapted from DINOV2 here: https://github.com/facebookresearch/dinov2
